dist/SyncTasks.js

```diff
@@ -116,8 +116,13 @@ var Internal;
             this._mustHandleError = true;
```

Comment:
We should stop publishing dist files on git like our other modules. Not needed in scope of this commit